### ISSUE TITLE
Incorporate Dependencies into Data Linkage with Cascading Changes

### ISSUE THREAD

**Issue by leoRysing** [Northeastern-Electric-Racing/FinishLine#231]
### Overview

Incorporate the linkage of dependencies in work packages to the changes to timelines with cascading changes. TBD what the solution should be.

### Stakeholders

Product Stakeholder:
Software Stakeholder:
Reference Users:

### User Story

As a _____, I want _____, so that _____.

### Success Metrics

Being able to measure user behaviors

### Rollout Plan

Integrate, collect, analyze

### Out of Scope

- 

### Background / Context

Dependencies are WBS #s listed and stored in Work Packages and represent items that must be completed before the current work package may begin. Currently, no aspect of FinishLine pays attention to dependencies as it relates to the interconnectedness of timelines.

### Acceptance Criteria & Mock-ups

List everything to be tracked:

### Tickets

 - [ ] #
 - [ ] #

**Comment by RChandler234** [comment]
Never got filled out
